modules/growthCodeRtdProvider.md

```diff
@@ -0,0 +1,55 @@
+## GrowthCode Real-time Data Submodule
```

Comment:
Please include a section explaining what is adjusting within the bidrequest and what targeting data will be added